docs/appliance-config.md

```diff
@@ -14,6 +14,12 @@
 | pullSecret                 |                                | No       | string  | PullSecret required for mirroring the OCP release payload.                                                                                                                             |     
 | sshKey                     |                                | Yes      | string  | Public SSH key for accessing the appliance during the bootstrap phase.                                                                                                                 |                 
 | userCorePass               |                                | Yes      | string  | Password of user 'core' for connecting from console.                                                                                                                                   |                        
-| imageRegistry              |                                | Yes      |         |                                                                                                                                                                                        |
+| imageRegistry              |                                | Yes      |         | Local image registry details (used when building the appliance)                                                                                                                                                                                       |
 | imageRegistry.uri          | `docker.io/library/registry:2` | Yes      | string  | The URI for the image. Alternative: `quay.io/libpod/registry:2.8`                                                                                                                      |                                                                                                   
 | imageRegistry.port         | 5005                           | Yes      | integer | The image registry container TCP port to bind. A valid port number is between 1024 and 65535.                                                                                          |                                                                                  
+| enableDefaultSources       | false                          | Yes      | bool    | Enable all default CatalogSources (on openshift-marketplace namespace). Should be disabled for disconnected environments.                                                                               
+|
+| additionalImages           |                                | Yes      | array   | Additional images to be included in the appliance disk image.
+|
+| operators                  |                                | Yes      | array   | Operators to be included in the appliance disk image. See examples in https://github.com/openshift/oc-mirror/blob/main/docs/imageset-config-ref.yaml.
+|
```

Comment:
copy this format:

| Name                       | Default Value                  | Optional | Type    | Description                                                                                                                                                                                                                       |
|----------------------------|--------------------------------|----------|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| apiVersion                 |                                | No       | enum    | The configuration version that is currently supported by the appliance. options: `v1beta1`.                                                                                                                                       |
| kind                       |                                | No       | string  | The configuration kind: `ApplianceConfig`.                                                                                                                                                                                        |
| ocpRelease                 |                                | No       |         |                                                                                                                                                                                                                                   |
| ocpRelease.version         |                                | No       | string  | OCP release version in `major.minor` or `major.minor.patch` format. In case of `major.minor` - latest patch version will be used. Note: if the specified version is not yet available, the latest supported version will be used. |                                                    
| ocpRelease.channel         | `stable`                       | Yes      | enum    | OCP release update channel: `stable`, `fast`, `eus`, `candidate`.                                                                                                                                                                 |          
| ocpRelease.cpuArchitecture | `x86_64`                       | Yes      | enum    | OCP release CPU architecture: `x86_64`, `aarch64`, `ppc64le`.                                                                                                                                                                     |                                                                           
| diskSizeGB                 |                                | Yes      | integer | Virtual size of the appliance disk image. If specified, should be at least 150GiB. Otherwise, the disk image should be resized when cloning to a device (e.g. using virt-resize tool).                                            |  
| pullSecret                 |                                | No       | string  | PullSecret required for mirroring the OCP release payload.                                                                                                                                                                        |     
| sshKey                     |                                | Yes      | string  | Public SSH key for accessing the appliance during the bootstrap phase.                                                                                                                                                            |                 
| userCorePass               |                                | Yes      | string  | Password of user 'core' for connecting from console.                                                                                                                                                                              |                        
| imageRegistry              |                                | Yes      |         | Local image registry details (used when building the appliance)                                                                                                                                                                   |
| imageRegistry.uri          | `docker.io/library/registry:2` | Yes      | string  | The URI for the image. Alternative: `quay.io/libpod/registry:2.8`                                                                                                                                                                 |                                                                                                   
| imageRegistry.port         | 5005                           | Yes      | integer | The image registry container TCP port to bind. A valid port number is between 1024 and 65535.                                                                                                                                     |                                                                                  
| enableDefaultSources       | false                          | Yes      | bool    | Enable all default CatalogSources (on openshift-marketplace namespace). Should be disabled for disconnected environments.                                                                                                         |
| additionalImages           |                                | Yes      | array   | Additional images to be included in the appliance disk image.                                                                                                                                                                     |
| operators                  |                                | Yes      | array   | Operators to be included in the appliance disk image. See examples in https://github.com/openshift/oc-mirror/blob/main/docs/imageset-config-ref.yaml.                                                                             |